src/hardware/pounder/mod.rs

```diff
@@ -329,12 +331,11 @@ impl AttenuatorInterface for PounderDevices {
     /// Reset all of the attenuators to a power-on default state.
     fn reset_attenuators(&mut self) -> Result<(), Error> {
         self.mcp23017
-            .digital_write(ATT_RST_N_PIN, false)
+            .write_gpio(mcp23017::Port::GPIOB, 0x0f)
             .map_err(|_| Error::I2c)?;
-        // TODO: Measure the I2C transaction speed to the RST pin to ensure that the delay is
-        // sufficient. Document the delay here.
+        // Duration of one I2C transaction is sufficiently long.
         self.mcp23017
-            .digital_write(ATT_RST_N_PIN, true)
+            .write_gpio(mcp23017::Port::GPIOB, 0x2f)
```

Comment:
I'd much rather we may some define for these constants. It's not ergonomic to look back at the schematic to figure out what 0x2F is supposed to correlate to (same for 0x0F above).

Reply:
Let's leave that for later. Magic constants matches the existing previous style (see e.g. L312 and L316).
This is about getting it to work now, not making it perfect.